geopandas/tests/test_sindex.py

```diff
@@ -670,6 +684,47 @@ def test_query_bulk_sorting(self, sort, expected):
                 )
             raise e
 
+    # ------------------------- `nearest` tests ------------------------- #
+    @pytest.mark.skipif(
+        TEST_NEAREST,
+        reason=(
+            f"PyGEOS >= {MIN_PYGEOS_VERSION_FOR_NEAREST}"
```

Comment:
```suggestion
            Pygeos >= 0.10 supports sindex.nearest
```

(note: this test is skipped when pygeos is _present_)